src/Microsoft.DocAsCode.MarkdigEngine.Extensions/MonikerRange/MonikerRangeBlock.cs

```diff
@@ -9,8 +9,12 @@ namespace Microsoft.DocAsCode.MarkdigEngine.Extensions
     public class MonikerRangeBlock : ContainerBlock
     {
         public string MonikerRange { get; set; }
+
+        public object ParsedMonikers { get; set; }
```

Comment:
how about just `Monikers`?

Reply:
That would confuse with `MonikerRange` because we have other places where `MonikerRange` and `Monikers` are used at the same level to represent input. This is not an input.